apps/minaintyg/src/pages/Certificates/components/CertificateCard/CertificateCardSummary.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react'
+import { CertificateListSummary } from '../../../../schema/certificateList.schema'
+import { CertificateCardSummary } from './CertificateCardSummary'
+
+it('Should render stuff', () => {
```

Comment:
Split up test in three tests. "Show render label" "Should render summary" "Should render date" instead of "Should render stuff"

Reply:
Added a couple of snapshot tests instead.